src/cli/commands/workspaces.js

```diff
@@ -27,8 +29,30 @@ export async function info(config: Config, reporter: Reporter, flags: Object, ar
   const publicData = {};
 
   for (const workspaceName of Object.keys(workspaces)) {
+    const {loc, manifest} = workspaces[workspaceName];
+
+    const workspaceDependencies = new Set();
+    const mismatchedWorkspaceDependencies = new Set();
+
+    for (const dependencyType of DEPENDENCY_TYPES) {
```

Comment:
Isn't it better to just omit this before the loop?

Reply:
What do you mean? Duplicate the DEPENDENCY_TYPES array, but without peerDependencies?

Reply:
Yes. It seems wasteful and less readable this way to check for this specific value in every iteration.

Reply:
Yeah, maybe we could have a `DIRECT_DEPENDENCY_TYPES` array.